### DISCUSSION TITLE
How to write stories using typescript for react native apps

### DISCUSSION THREAD

**Discussion by suprith-s-reddy** [storybookjs/react-native#191] [General]
Hello all,

I would like to understand if there is a possibility to write the stories file in tsx for the react-native storybook. Have any of you tried it?
If so please let me know and kindly share an example in the comments.

**Comment by dannyhw** [comment]
Yes typescript is useable just like with any other react native code. Is there anything specifically you are struggling with?

**Comment by suprith-s-reddy** [comment]
No thanks, I figured it out myself. Currently the storybook does not work
on the web for react native. Over that I tried .ts and I messed it. Now I
have cleaned it and ts generally works. But the storybook for react-native
web is not working. Need to find a solution for it.

**Reply by dannyhw** [reply]
If you downgrade the react-native-web package to 0.11.7 it should work. There will be a fix in 6.0 though.
